src/math/unary_ops.ts

```diff
@@ -121,8 +122,14 @@ export class Ops {
         (min <= max),
         `Error in clip: min (${min}) must be` +
             `less than or equal to max (${max}).`);
-    return ENV.engine.executeKernel('Clip', {inputs: {x}, args: {min, max}}) as
-        T;
+    return ENV.engine.executeKernel(
```

Comment:
`dl.where` currently does not support `Scalar` values. We should throw an error for this.

Reply:
Done.